src/index.ts

```diff
@@ -91,10 +111,12 @@ module.exports = function(app) {
         app.error("Error:" + subscriptionError);
       },
       (delta) => {
+        // @ts-ignore: todo: fix Delta type upstream
         delta.updates.forEach(({ values }) => {
-          values.forEach(({ path, value }) => {
+          // @ts-ignore: todo: fix Delta type upstream
+          values.forEach(({ path }) => {
```

Comment:
FIXME: `delta` is missing types. These can be generated from the JSON Schema using a tool like [json2ts](https://www.npmjs.com/package/json2ts), but is going to require some mapping for paths to schemas.